scripts/lib.sh

```diff
@@ -46,29 +48,67 @@ query-apple-m1() {
 }
 
 get-pyenv-version() {
+    if [[ -n "$SENTRY_PYTHON_VERSION" ]]; then
+        echo "${SENTRY_PYTHON_VERSION}"
+        return 0
+    fi
+
     local PYENV_VERSION
     PYENV_VERSION=3.6.13
     if query-apple-m1; then
-        PYENV_VERSION=3.8.11
+        PYENV_VERSION=3.8.12
     fi
     echo "${PYENV_VERSION}"
 }
 
 query-valid-python-version() {
+    if [[ -n "$SENTRY_PYTHON_VERSION" ]]; then
+        python_version=$(python3 -V 2>&1 | awk '{print $2}')
+        if [ "$python_version" != "$SENTRY_PYTHON_VERSION" ]; then
+            cat <<EOF
+    ${red}${bold}
```

Comment:
Why Why does this have the whitespace at the front?

Reply:
lol